tools/ipapy2/ipapy2_compute_bio.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import os
+import pandas as pd
+from ipaPy2 import ipa
+
+
+def main(args):
+    MS1_DB = pd.read_csv(args.MS1_DB)
+    MS1_DB = MS1_DB.replace("", None)
+
+    if args.annotations:
+        annotations_df = pd.read_csv(args.annotations, keep_default_na=False)
+        annotations_df = annotations_df.replace("", None)
+        annotations = {}
+        keys = set(annotations_df["peak_id"])
+        for i in keys:
+            annotations[i] = annotations_df[annotations_df["peak_id"] == i].drop(
+                "peak_id", axis=1
+            )
+    else:
+        annotations = None
```

Comment:
are the annotations optional for this tool?

Reply:
Yeah, the annotations are optional and could be set to None.